Re-seed weights in IaC concepts

jkodroff opened the pull request:
This should be a no-op from the perspective of the user - the menu items order should remain the same, but it will make future reordering easier.

thoward commented on content/docs/iac/concepts/secrets.md:
````diff
@@ -220,7 +220,7 @@ By default, the Pulumi CLI uses a per-stack encryption key managed by Pulumi Clo
 For example, this command sets a configuration variable named `dbPassword` to the plain-text value `S3cr37`:
 
 ```bash
-$ pulumi config set --secret dbPassword S3cr37
````
I suggest that we adopt the Google style guide recommendations here: https://developers.google.com/style/code-syntax#prompt ... basically, showing the `$` prompt is always preferred (it's technically optional for single commands, but required for multiple commands in the same listing). 

In general, I consider the Google style guide our base style, with some Pulumi-specific variations that override it.

jkodroff replied:
Markdownlint says you only include the `$` if you show the output. I'm totally cool if we change this with a global config, though.

jkodroff commented:
This is now fixed. We have to disable format on save in VSCode to prevent this from happening in the future, so I've added a project settings file to fix.

The other files were also restored, save for the intended changes.
